node/src/components/small_network.rs

```diff
@@ -793,12 +818,14 @@ where
 
     /// Updates a set of known joining nodes.
     /// If we've just connected to a non-joining node that peer will be removed from the set.
-    fn update_joining_set(&mut self, peer_id: NodeId, is_joiner: bool) {
-        // Update set of joining peers.
-        if is_joiner {
-            self.joining_nodes.insert(peer_id);
+    fn update_syncing_nodes_set(&mut self, peer_id: NodeId, is_syncing: bool) {
+        // Update set of syncing peers.
+        if is_syncing {
+            info!(%peer_id, "is syncing");
```

Comment:
Please lower these to debug level, otherwise SRE will hunt me down and make me fix it.

Reply:
Done in 5b91f0f74616a4c60822d297c58f08523917ec38